nav2_behavior_tree/include/nav2_behavior_tree/bt_action_server_impl.hpp

```diff
@@ -59,6 +60,24 @@ BtActionServer<ActionT>::BtActionServer(
   if (!node->has_parameter("default_server_timeout")) {
     node->declare_parameter("default_server_timeout", 20);
   }
+
+  std::vector<std::string> error_code_id_names = {
+    "follow_path_error_code_id",
```

Comment:
I think these should be `follow_path_error_code` without the ID. The ID is the identifier for the code (e.g. `error_code_id` port to be populated about what ID the error code is stored in). The actual error code storage variable shouldn't have ID.

Please fix here + BTs + configuration file